app/views/jobseekers/profiles/show.html.slim

```diff
@@ -0,0 +1,13 @@
+- content_for :page_title_prefix, t(".page_title")
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1.govuk-heading-xl = t(".page_title")
+    p = t(".page_description")
+
+    - @sections.each do |section|
+        h2.govuk-heading-m class="govuk-!-font-size-27 govuk-!-padding-top-6"
```

Comment:
seems a bit odd the have `govuk-!-font-size-27` class when `govuk-heading-m` is size 24 and the standard govuk size?

Reply:
Good spot